src/pages/Article/ArticlePage.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PageLayout from '../../components/PageLayout';
+import Button from "@mui/material/Button";
+import { Box } from '@mui/system';
+
+/**
+ * Renders the Article page.
+ * @returns {JSX.Element} ArticlePage component
+ */
+function ArticlePage() {
+  return (
+	<PageLayout viewportPage title='Article'>
+		<Box sx={{ 
+			display: "flex",
+			flexDirection: "column",
+			justifyContent: "center",
+			alignItems: "center",
+			height: "calc(100% - 60px)"
+			}}>
+
+
+			{/* box where the article is displayed */}
```

Comment:
I think we can delete this comment here, because it is obvious that he Box holds the Article

Reply:
done